src/Router.js

```diff
@@ -1,13 +1,32 @@
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { useState, useEffect } from 'react';
 
 import Main from './pages/Main/Main';
+import Nav from './components/Nav/Nav';
 
 export default function Router() {
+  const [windowY, setWindowY] = useState();
+  const [visibleValue, setVisibleValue] = useState(true);
+
+  useEffect(() => {
+    function handleScroll() {
+      windowY > window.scrollY ? setVisibleValue(true) : setVisibleValue(false);
```

Comment:
```suggestion
      sthBlahBlah ? setVisibleValue(true) : setVisibleValue(false);
```
조건식 그대로 두기보다 좋은 변수명 있다면 이름 지어주는게 좋을 것 같아요

Reply:
저번에 예시들어주신 것처럼 isScrollUp 이라는 boolean 변수로 제어하도록 만들었습니다.